### PR TITLE
#365 | NFT preview component

### LINKED ISSUE #365: implement NFT Preview component
- NFT preview should be displayed. the types are as follows: 
    - include appropriate accessibility html, aria and title 
    - no media
        - display gray background with broken image icon (material icons - image-not-supported) 
    - image
        - display image
    - video
        - show cover image with play icon in center (material icons - play arrow)
        - if a video NFT does not supply a cover image, use the first frame as the image
    - audio with cover image
        - show cover image with headphones icon in the center (material outline icons - headphones)
    - audio without cover image
        - show gray tile with headphones icon
- should display the NFT name and ID
- should display a link to the NFT collection owner address (or name, if we can get it) which links to teloscan
- add this component to the demos under a new route like NFTs

### PR DESCRIPTION
**Pull request by EJSG**
# Fixes #365

## Description
This PR adds an NFT preview component, for use on the inventory page

## Test scenarios
- go to
- log in
- click the new NFT link
    - you should see a barebones page with NFT previews
    - the various types of NFT should display according to the acceptance criteria
- change the size of your screen progressively down to mobile
    - the page should be responsive
- click on an NFT
    - you should be directed to the detail page with the correct URL params (contract and ID)
-  go back
- click on the external link under the nft
- you should be taken to the correct teloscan page 

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file and/or created a new issue with the required translations for the currently supported languages
-   [x] I have tested for mobile functionality and responsiveness

### CODE REVIEW

File: src/pages/evm/nfts/NftViewer.vue [hunk 1]
```diff
@@ -0,0 +1,207 @@
+<script setup lang="ts">
+/*
+Possible NFT configurations after shaping:
```
Comment (EJSG) [inline src/pages/evm/nfts/NftViewer.vue:3]: this can all be removed after the component is complete (after nft detail page)

**Review by karynemayer — COMMENTED** [review]
This looks great! The grid is perfect!
I have two tiny feedbacks:
1) How would you feel about adding some transparency to the icon background?
Screenshot 2023-06-08 at 14 23 34
The image there I changed the circle background from `#FFFFFF` to `#FFFFFFA0`. I feel it helps to keep the contrast adequately and doesn't block the image entirely.

2) For those beautiful people with SE phones, we have a horizontal scroll. The size of the item matches the screen width, but we still have some margins on the right side that makes it scrollable. Also, the margin on the left is 0. 

Screenshot 2023-06-08 at 14 15 41


Personally, I would love to keep some margin even in the tiniest of screens, but if thats too much I'm fine with just removing the right margin.

File: src/i18n/en-us/index.js [hunk 2]
```diff
@@ -116,6 +118,11 @@ export default {
         rows_per_page: 'Rows per page',
         amount: 'Amount',
     },
+    nft : {
+        link_to_nft_details: 'Go to NFT details page for {name}',
+        img_alt: 'NFT - {nftInfo}',
+        img_alt_video_nft: 'Video NFT cover - {nftInfo}',
+    },
```
Comment (Viterbo) [inline src/i18n/en-us/index.js:125]:
```suggestion
    nft : {
        link_to_nft_details: 'Go to NFT details page for {name}',
        img_alt: 'NFT - {nftInfo}',
        img_alt_video_nft: 'Video NFT cover - {nftInfo}',
        details_page_title: 'NFT Details Page',
        tile_toggle_radio_label: 'Show as tile?'
    },
```

File: src/pages/evm/nfts/NftInventoryPage.vue [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+</script>
+
+<template>
+<AppPage>
+    <template v-slot:header>
+        <h1>{{ $t('evm_wallet.inventory') }}</h1>
+    </template>
+
+    <div class="c-nft-page">
+        <q-checkbox v-model="showNftsAsTiles" class="q-mb-lg">Show as tile?</q-checkbox>
```
Comment (Viterbo) [inline src/pages/evm/nfts/NftInventoryPage.vue:190]:
```suggestion
        <q-checkbox v-model="showNftsAsTiles" class="q-mb-lg">{{ $t('nft.tile_toggle_radio_label') }}</q-checkbox>
```

File: src/pages/evm/nfts/NftViewer.vue [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    max-height: 270px;
+
+    &__image-container,
+    &__image,
+    &__video-container,
+    &__audio-container,
+    &__blank-container {
+        height: 100%;
+        width: 100%;
+    }
```
Comment (Viterbo) [inline src/pages/evm/nfts/NftViewer.vue:150]: I really love this way of styling things!
I learned it by reviewing your code and now is part of me :smiley:

**Review by Viterbo — APPROVED** [review]
Really nice work! Ez, you're awesome!! :muscle:

I left some minor suggestions for changes to do.

Also, great job on defining those interfaces !!
I will make the NFT class implement that interface so we don't have to change anything you've just done.

**Review by donnyquixotic — APPROVED** [review]

File: src/i18n/en-us/index.js (hunk 2 above)
Comment (EJSG) [reply src/i18n/en-us/index.js]: this will need to be removed in the future because this text will not actually be on the page btw

**Review by pmjanus — APPROVED** [review]

**Pull request merged**